### ISSUE TITLE
Auto-reply for email

### ISSUE THREAD

**Issue by YedidaZalik** [cds-snc/notification-planning#2908]
The confirmation message that you get after emailing the a11y feedback inbox will need to be set up as an auto-reply by whoever manages that inbox as Steve does not have access to it

Figma content for the auto-reply:

- [x] [EN](https://www.figma.com/design/f6QILuxf94EHaStf2ejWJb/Contact-us-%7C-Nous-joindre?node-id=2120-3612&m=dev&focus-id=2120-2546)

- [x] [FR](https://www.figma.com/design/f6QILuxf94EHaStf2ejWJb/Contact-us-%7C-Nous-joindre?node-id=2120-2618&m=dev&focus-id=2120-2546)
